### PR TITLE
Pull request for 17-homepage

### LINKED ISSUE #17: Finish homepage
Design [aici](https://drive.google.com/drive/u/0/folders/0B4SZODI8XCNzVGFXTlhrVDBJR2c/0B_wU4FGfmo2ZbDhRVnJvd09PMXc/0B0dB3zP2ctjaT0x6WG1Sa2VLd2c).
- [ ] solutie pentru cum fac layouturile. Robert sugerase cu min-height pe div-uri care au background-image/color.
- [ ] bucata verde
- [ ] bucata gri
- [ ] bucata galbena
- [ ] sponsori
- [ ] footer

### PR DESCRIPTION
**Pull request by palcu**
Fixes #17

### CODE REVIEW

File: src/main.less [hunk 1]
```diff
@@ -9,71 +9,93 @@
...
+
+      .values td {
+        font-size: 34pt;
+        color: @orange;
+      }
     }
   }
+
+  .green-section {
+    background-color: green;
```
Comment (vtemian) [inline src/main.less:89]: Nu ar fi bine sa folosim @green?

**Comment by robertdxyz** [comment]
screenshot from 2015-05-18 15 54 50
screenshot from 2015-05-18 15 55 11

When the title extends to a new line the background image is zoomed and it creates an unusual effect. Can we make it have the same zoom level?

The menu should transform into a burger menu.

File: src/main.less (hunk 1 above)
Comment (palcu) [inline src/main.less:89]: Sunt doar placeholders. O sa pun culorile care trebuie. Eu intrebam daca legat de layout am facut bine sau aveti idei mai bune. Va trebui sa facem cate un `inner-row` pentru fiecare row de acum incolo.

Comment (palcu) [inline src/main.less:89]: Nop, e futut pe mobil cum am vrut eu sa fac. E hate CSS.

**Comment by robertdxyz** [comment]
The logo should never disappear. The menu should become hamburger and the logo should stay there.

**Comment by vtemian** [comment]
Good enough for now!

**Pull request merged**